include/metavision_driver/metavision_wrapper.h

```diff
@@ -60,6 +60,15 @@ class MetavisionWrapper
     size_t maxQueueSize{0};
   };
 
+  struct TrailFilter
+  {
+    enum Type { TRAIL, STC_CUT_TRAIL, STC_KEEP_TRAIL };
```

Comment:
This enum is not needed. Keep type as a string, and use std::map to map to the metavision SDK enums at the very last moment.